src/ibex_bluesky_core/devices/simpledae/strategies.py

```diff
@@ -0,0 +1,59 @@
+"""Base classes for DAE strategies."""
+
+from typing import TYPE_CHECKING
+
+from ophyd_async.core import (
+    Device,
+)
+
+if TYPE_CHECKING:
+    from ibex_bluesky_core.devices.simpledae import SimpleDae
+
+
+class ProvidesExtraReadables:
+    """Strategies may specify interesting DAE signals using this method.
+
+    Those signals will then be added to read() and describe() on the top-level SimpleDae object.
+    """
+
+    def additional_readable_signals(self, dae: "SimpleDae") -> list[Device]:
```

Comment:
This function, as is, could be static - but maybe the intent is to overload it, then 'self' might become significant.